controller/integration_tests/solenoid_test.h

```diff
@@ -11,8 +11,7 @@ void run_test() {
   bool solenoid_state = false;
 
   while (true) {
-    Hal.digitalWrite(BinaryPin::SOLENOID,
-                     solenoid_state ? VoltageLevel::LOW : VoltageLevel::HIGH);
+    Hal.PSOL_Value(solenoid_state ? 0.0f : 1.0f);
```

Comment:
Could we add a TODO to this file to update it to the fact that the binary solenoid it talks about is in fact a proportional solenoid?